beacon.lua

```diff
@@ -1,12 +1,16 @@
+local S = minetest.get_translator("telemosaic")
+local NS = function(s) return s end
```

Comment:
It is meant to help the [translation updater tool](https://github.com/minetest/minetest/blob/5.8.0/util/README_mod_translation_updater.md#how-to-use-ns) find strings that are fed into arguments to S.

Either I should remove it, because I did not end up using the tool and haven't marked everything with NS that should be, or else I will need to refactor more code to insert NS calls. I am in favour of removing it at the moment; I don't see how the translation updater script will help unless telemosaic gets new content  and updating the translation(s) is seen as onerous.

Reply:
Yeah, remove it.